src/backgrounds.jl

```diff
@@ -0,0 +1,216 @@
+"""
+
+`draw_grid(video::Video, action::Action, frame::Int; direction::Symbol = :TR, line_gap::Int = 25)`
+
+Draws an oriented grid on the given frame of a Video.
+
+# Arguments
+- `direction::Symbol`: Where grid animation finishes. Default: `:TR` Available Orientations:
+  - `:TR` - Animation finishes in the **T**op **R**ight corner of the frame.
+  - `:TL` - Animation finishes in the **T**op **L**eft corner of the frame.
+  - `:BR` - Animation finishes in the **B**ottom **R**ight corner of the frame.
+  - `:BL` - Animation finishes in the **B**ottom **L**eft corner of the frame.
+- `line_gap::Int`: How many pixels between each line. Default: `25`
+
+# Example
+This example will produce a grid animation that finishes in the top left corner of the animation. One will need to define their own path for `tempdirectory` and `pathname`.
+
+```
+using Javis
+using Luxor
+
+function ground(args...)
+    background("white")
+    sethue("black")
+end
+
+video = Video(500, 500)
+javis(
+      video,
+      [
+          Action(1:100, ground),
+    	  Action(1:100, :line, (video, action, frame)-> draw_grid(video, action, frame, direction = :TL, line_gap = 25)),
+      ],
+      tempdirectory = "/tmp-directory",
+      creategif = true,
+      pathname = "/tmp-directory/grid_animation.gif"
+     )
+
+```
```

Comment:
I think it's quite tedious to write an example like this every time so maybe we want to just give an example:
```
...
 Action(1:100, :line, draw_grid(direction = :TL, line_gap = 25))
...
```
where `...` can be looked up in the `javis` function. (This would be the example after closures are implemented)

Reply:
Fair, I agree with that. I liked the idea of having examples in the docstring simply to give a minimal working example of how it is to function. But perhaps this would be better suited to a tutorial (ping #25)